src/components/maybe-internal-link/index.tsx

```diff
@@ -1,18 +1,17 @@
+import React from 'react'
 import Link from 'next/link'
 import isAbsoluteUrl from 'lib/is-absolute-url'
-import React from 'react'
+import { MaybeInternalLinkProps } from './types'
```

Comment:
Split this out into a `types.ts` file for consistency (and to remove the "arg should be typed" warning in `InternalLink` within this file).